lib/response.js

```diff
@@ -795,7 +795,7 @@ res.get = function(field){
  */
 
 res.clearCookie = function clearCookie(name, options) {
-  var opts = merge({ expires: new Date(1), path: '/' }, options);
+  var opts = options? merge({ expires: new Date(1), path: '/' }, options): { expires: new Date(1), path: '/' };
```

Comment:
Can this be done without having to maintain two copies of the same object? That would seem to decrease the maintainability, which seems contrary to the goal of the pr.

Reply:
I have put the same object in a shared variable

Reply:
Can you do that for all the other ones that are like this? Even an empty object twice should be obvious that those are the same for maintainabibility so one doesn't get changed and not the other.